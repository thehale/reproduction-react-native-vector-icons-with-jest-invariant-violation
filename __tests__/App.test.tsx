/**
 * @format
 */

import 'react-native';

import App, { Icon } from '../App';
// Note: import explicitly to use the types shipped with jest.
import {expect, it} from '@jest/globals';

import React from 'react';
// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  renderer.create(<App />);
});

it('renders icons correctly', () => {
  let tree = renderer.create(<Icon />).toJSON();
  expect(tree).toMatchSnapshot();
})