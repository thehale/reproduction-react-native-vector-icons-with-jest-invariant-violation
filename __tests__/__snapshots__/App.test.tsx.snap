// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders icons correctly 1`] = `
<icon
  color="#900"
  name="flag"
  size={30}
/>
`;
